pkgs/tools/package-management/lix/default.nix

```diff
@@ -188,7 +202,51 @@ lib.makeExtensible (self: {
     };
   };
 
-  latest = self.lix_2_92;
+  lix_2_93 = self.makeLixScope {
+    attrName = "lix_2_93";
+
+    lix-args = rec {
+      version = "2.93.0";
+
+      src = fetchFromGitea {
+        domain = "git.lix.systems";
+        owner = "lix-project";
+        repo = "lix";
+        rev = version;
+        hash = "sha256-hsFe4Tsqqg4l+FfQWphDtjC79WzNCZbEFhHI8j2KJzw=";
+      };
+
+      cargoDeps = rustPlatform.fetchCargoVendor {
+        name = "lix-${version}";
+        inherit src;
+        hash = "sha256-YMyNOXdlx0I30SkcmdW/6DU0BYc3ZOa2FMJSKMkr7I8=";
+      };
+    };
+  };
+
+  git = self.makeLixScope {
+    attrName = "git";
+
+    lix-args = rec {
+      version = "2.94.0-pre-20250509_${builtins.substring 0 12 src.rev}";
+
+      src = fetchFromGitea {
+        domain = "git.lix.systems";
+        owner = "lix-project";
+        repo = "lix";
+        rev = "dcb0a97000d50b2868ed4f8d9fd465c5a5b8eb3a";
+        hash = "sha256-qCRBy8Bbh5XhPalPkhonxNgfsbw3lP0UIXBLSrhxAvI=";
+      };
+
+      cargoDeps = rustPlatform.fetchCargoVendor {
+        name = "lix-${version}";
+        inherit src;
+        hash = "sha256-YMyNOXdlx0I30SkcmdW/6DU0BYc3ZOa2FMJSKMkr7I8=";
+      };
+    };
+  };
+
+  latest = self.lix_2_93;
```

Comment:
Should we set this as stable too? AFAIK there are no upstream or third party blockers like 2.92, and @RaitoBezarius signaled towards skipping over that release anyways [here](https://github.com/NixOS/nixpkgs/pull/405718#issuecomment-2869961790)

Reply:
I do think we have one major regression, which is the SSH wrapper stuff, atm, that I will be trying to fix in https://gerrit.lix.systems/c/lix/+/3159 and backport it. cc @lf-